src/content/translations/id/developers/docs/intro-to-ether/index.md

```diff
@@ -0,0 +1,76 @@
+
+## Mencetak ether {#minting-ether}
+
+Mencetak adalah proses di mana ether baru dibuat di buku besar Ethereum. Protokol Ethereum yang mendasarinya membuat ether baru, dan ini tidak memungkinkan untuk seorang pengguna membuat ether.
+
+Ether dicetak saat seorang penambang membuat sebuah blok di blockchain Ethereum. Sebagai insentif bagi para penambang, protokol memberikan imbalan di setiap blok, yang menambahkan saldo dari alamat yang ditetapkan oleh blok penambang. Nilai imbalan blok telah berubah dari waktu ke waktu, dan hari ini nilainya sebesar 2 ETH per blok.
+
+## Membakar ether {#burning-ether}
+
+Sama halnya dengan pembuatan ether melalui imbalan blok, ether dapat dihancurkan oleh proses yang disebut 'pembakaran'. Ketika ether dibakar, ether dihilangkan dari perederan secara permananen.
+
+Pembakaran ether terjadi dalam setiap transaksi di Ethereum. Ketika para pengguna membayar transaksi mereka, biaya gas dasarnya dihancurkan oleh protokol. Tergantung pada permintaan jaringan, [beberapa blok](https://etherscan.io/block/12965263) membakar lebih banyak ether ketimbang mencetaknya.
+
+## Denominasi ether {#denominations}
+
+Karena banyak transaksi di Ethereum bernilai kecil, ether memiliki beberapa denominasi yang mungkin dirujuk untuk jumlah nilai yang lebih kecil. Dari denominasi ini, Wei dan gwei sangat penting.
+
+Wei adalah jumlah ether terkecil, dan sebagai hasilnya, banyak implementasi teknis, seperti [Ethereum Yellowpaper](https://ethereum.github.io/yellowpaper/paper.pdf), akan mendasarkan semua penghitungan dalam Wei.
+
+Gwei, kependekan dari giga-wei, sering digunakan untuk menggambarkan biaya gas di Ethereum.
+
+| Denominasi | Nilai dalam ether | Penggunaan Umum                |
+| ---------- | ----------------- | ------------------------------ |
+| Wei        | 10<sup>-18</sup>  | Implementasi teknis            |
+| Gwei       | 10<sup>-9</sup>   | Biaya gas yang terbaca manusia |
+
+## Mentransfer ether {#transferring-ether}
+
+Setiap transaksi di Ethereum berisi bidang `nilai`, yang menentukan jumlah ether yang akan ditransfer, dalam denominasi wei, untuk dikirim dari alamat pengirim ke alamat penerima.
+
+Ketika alamat penerima merupakan [kontrak pintar](/developers/docs/smart-contracts/), ether yang ditransfer ini dapat digunakan untuk membayar gas ketika kontrak pintar menjalankan kodenya.
+
+[Lebih lanjut tentang transaksi](/developers/docs/transactions/)
+
+## Membuat kueri ether {#querying-ether}
+
+Pengguna dapat membuat kueri saldo eter dari [akun](/developers/docs/accounts/) dengan memeriksa bidang `saldo` akun, yang menunjukkan kepemilikan ether dalam mata uang wei.
```

Comment:
Inline code snippets should not be translated. These are in code properties within an Ethereum transaction

```suggestion
Pengguna dapat membuat kueri saldo eter dari [akun](/developers/docs/accounts/) dengan memeriksa bidang `balance` akun, yang menunjukkan kepemilikan ether dalam mata uang wei.
```